test/integration/servicecluster.go

```diff
@@ -148,7 +148,7 @@ func newServiceClusterSuite(
 			},
 		}
 		require.NoError(t, managementClient.Create(ctx, catalogEntrySet))
-		require.NoError(t, testutil.WaitUntilReady(ctx, managementClient, catalogEntrySet))
+		require.NoError(t, testutil.WaitUntilReady(ctx, managementClient, catalogEntrySet), testutil.WithTimeout(time.Minute))
```

Comment:
how bout making a dedicated integration test suite for the `CustomResourceDiscovery`? Similar to how `DerivedCustomResource` has its own dedicated suite. Especially since we're going to test more for this integration suite than what should be in scope for the service cluster.

Reply:
but if you look into the servicecluser suite closely, it is integration test suite for `CustomResourceDiscovery`,  it is called `servicecluster` because it is doing some cross-cluster tests IMO.

Reply:
ok; seems good to me at least for this PR.